Zeplin.sketchplugin/Contents/Sketch/utils.cocoascript

```diff
@@ -312,6 +321,10 @@ var exportArtboards = function (context, artboards, temporaryPath) {
         [directives setObject:shareId forKey:@"shareId"];
     }
 
+    if (sourceFileName) {
+        [directives setObject:sourceFileName forKey:@"sourceFileName"];
+    }
+
     shareId = nil;
```

Comment:
After setting the `sourceFileName` into the directives we should set it `nil` as we did for others.

Reply:
✅